src/input/touch.c

```diff
@@ -133,6 +145,8 @@ handle_touch_down(struct wl_listener *listener, void *data)
 			}
 		}
 
+		wlr_cursor_warp_absolute(seat->cursor,
```

Comment:
Note to self, look briefly if this behaves with multiple touch points, thus when more than one finger is down.

Reply:
Answer: the cursor does jump back and forth between two finger when moving both. But this is only visible when not hiding the cursor, so probably not an issue.

(I have a branch that adds a "hideCursor" option for touch/tablet for testing, not sure yet how useful this is).

Reply:
The last commit addresses this and warps the cursor only when exactly one finger is down.